services/github/src/handlers/status.js

```diff
@@ -177,10 +196,13 @@ export async function statusHandler(message) {
         taskDefinition.payload?.logs?.live || taskDefinition.payload?.log,
       ),
     ));
+    output.addText(markdownAnchor(CHECKTASK_TEXT, taskGroupUI(this.context.cfg.taskcluster.rootUrl, taskGroupId)));
+
     if (customCheckRunText) {
       output.addText(customCheckRunText);
     }
     if (liveLogText) {
+      output.addText(markdownLog(headLog(liveLogText, 20, githubCheck.output.getRemainingMaxSize())));
```

Comment:
I think it would be good to add some separator between both lines to make it clear that something got cut off from the middle
Or, maybe, it would be better to have single function that would take this `liveLogText`, split it into lines, then grab first `20` and last `~200` lines and adds `...(X lines hidden)...` 
Would be easier to observe `getRemainingMaxSize()` in single call

Just a suggestion, thanks!

Reply:
![Screenshot 2024-08-28 at 12 00 54 AM](https://github.com/user-attachments/assets/34e9d77d-c5fe-4ee1-8206-333e4b5ab60c)

In my local dev setup I have logged 10 lines of head, 10 lines of tail and hidden in between, will change these values while pushing.

Reply:
looks great!